src/Command/ScanFileByUDICommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+use App\Entity\DatasetSubmission;
+use App\Entity\Dataset;
+use App\Message\VirusScan;
+
+/**
+ * This command triggers a file virus scan, by UDI.
+ */
+class ScanFileByUDICommand extends Command
```

Comment:
I think we should remove this command. Manual scans can just be done by other means.

Reply:
If we remove it, we need a way to get a listing of file paths and names to all files associated with an UDI. Mass-scanning the storage directory would also require a means to back-resolve a file to an UDI.

Reply:
There a way to look it up I guess. I just don't see this command ever being used. Scan once in ingress, maybe again on egress. If period scans and required, we can deal with it then.

Reply:
I'd like to leave this in for ease of testing until we hook it into something else. It's otherwise hard to actually test a run of it.